package.json

```diff
@@ -14,7 +14,6 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^27.5.2",
-    "@types/node": "^16.18.11",
```

Comment:
Was this removed on purpose? Do we need this?

Reply:
Hm I think I unintentionally removed this trying to follow some guides on stack overflow. If we had it originally, I'll bring it back